projects/js-packages/components/components/jetpack-footer/index.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import A8cSvgTitle from '../a8c-svg-title';
+import './style.scss';
+
+/**
+ * JetpackFooter component definition.
+ *
+ * @param {object} props - Component properties.
+ * @param {object} props.aboutPageUrl - Link for 'An Automattic Airline'.
+ * @param {object} props.moduelName - Name of the moduel, e.g. 'Jetpack Search'.
```

Comment:
Maybe worth fixing that typo (moduel -> module) here and below?

Reply:
fixed in 38f33d9